server/repository/src/db_diesel/clinician_store_join_row.rs

```diff
@@ -21,7 +21,6 @@ table! {
 }
 
 joinable!(clinician_store_join -> clinician (clinician_id));
-allow_tables_to_appear_in_same_query!(clinician, clinician_store_join);
```

Comment:
Think its better to leave it here instead having one big list of all tables. Quite a while ago I looked into it and if I remember correctly having all tables in one macros generates way too much code and explodes if there are too many combinations...

See bottom of:
https://docs.rs/diesel/latest/diesel/macro.allow_tables_to_appear_in_same_query.html

Reply:
[5264f8a](https://github.com/openmsupply/open-msupply/pull/2035/commits/5264f8af85f1dc19434263ecff9b1216805ce23a)